specification/security/resource-manager/Microsoft.Security/stable/2024-08-01/securityStandards.json

```diff
@@ -0,0 +1,391 @@
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a security standard over subscription scope": {
+            "$ref": "./examples/SecurityStandards/DeleteBySubscriptionSecurityStandard_example.json"
+          },
+          "Delete a security standard over security connector scope": {
+            "$ref": "./examples/SecurityStandards/DeleteBySecurityConnectorSecurityStandard_example.json"
+          },
+          "Delete a security standard over management group scope": {
+            "$ref": "./examples/SecurityStandards/DeleteByManagementGroupSecurityStandard_example.json"
+          }
+        },
+        "tags": [
+          "SecurityStandards"
+        ],
+        "description": "Delete a security standard over a given scope",
+        "operationId": "SecurityStandards_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/StandardId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - security standard was deleted"
+          },
+          "204": {
+            "description": "No Content - security standard does not exist"
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityStandardList": {
+      "type": "object",
+      "description": "Page of a Standard list",
+      "properties": {
+        "value": {
+          "description": "Collection of standards in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityStandard"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "SecurityStandard": {
+      "type": "object",
+      "description": "Security Standard on a resource",
+      "properties": {
+        "properties": {
+          "description": "Properties of a security standard",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StandardProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "StandardProperties": {
+      "type": "object",
+      "description": "Describes properties of a standard.",
+      "properties": {
+        "displayName": {
+          "description": "display name of the standard, equivalent to the standardId",
```

Comment:
>d [](http://example.com/codeflow?start=26&length=1)

nit: Could you standardize starting with Capital letter. It is inconsistent throughout all swagger files.

Reply:
Fixed.